### PR TITLE
Added the ocs-remote template

### PR DESCRIPTION
**Pull request by Shirisha-S-Rao-1**

### CODE REVIEW

**Comment by Shirisha-S-Rao-1** [comment]
Results of testing this template captured here : [OCS-remote test results](https://github.ibm.com/alchemy-containers/armada-storage/issues/1876)

**Review by akgunjal — APPROVED** [review]
LGTM

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+2. Create HMAC credentials. Make a note of your credentials.
+    ```
+    $ ibmcloud resource service-key-create cos-cred-rw Writer --instance-name noobaa-stor --parameters '{ "HMAC": true}'
+    ```
+
+3. Get the regional public endpoint and the location of your IBM COS instance.
+    You can get the endpoint details from the list of [IBM COS endpoints](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints) and the
+    location from [IBM COS location constraint details](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-classes).
+
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:32]:
```suggestion
3. Get the regional public endpoint and the location of your IBM COS instance.
    You can get the endpoint details from the list of [IBM COS endpoints](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints) and the location from [IBM COS location constraint details](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-classes).

```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+| Parameter | Required? | Description | Default value if not provided | Datatype |
+| --- | --- | --- | --- | --- |
+| `ocs-cluster-name` | Required | Enter the name of your OcsCluster custom resource. | N/A | string |
+| `mon-storage-class` | Required | Enter the storage class name that you want to use for the mon pods | N/A | string |
+| `mon-size` | Required | Enter the size of the mon pods | 20Gi | string |
+| `osd-storage-class` | Required | Enter the storage class name that you want to use for the osd pods | N/A | string |
+| `osd-size` | Required | Enter the size of the osd pods | 100Gi | string |
+| `num-of-osd` | Optional | Enter the number of OSDs. OCS will create 3x number of OSDs for the value specified. Initial storage capacity is the same as your osd size specified at `osd-size`. When you want to increase your storage capacity, you have to increase `num-of-osd` by the multiples of `osd-size` | 1 | integer |
+|`worker-nodes` | Optional |Workers which need to be a part of OCS (Minimum 3). | N/A |csv |
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:45]:
```suggestion
|`worker-nodes` | Optional | Enter the IP addresses of the worker nodes where you want to deploy OCS. If you do not specify the `worker-nodes`, OCS is installed on all of the worker nodes in your cluster. The minimum number of worker nodes that you must specify is 3. | N/A |csv |
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+| sat-ocs-cephfs-gold | cephfs |  N/A | N/A | N/A | SSD | Delete |
+| sat-ocs-cephrgw-gold | ceph-rgw | N/A | N/A | N/A | SSD |Delete |
+| sat-ocs-noobaa-gold | noobaa |  N/A | N/A | N/A | N/A | Delete |
+
+
+
+
+## Creating the Red Hat Openshift Container Storage - Remote storage configuration
+
+1. Log in into the Cluster using oc CLI or IBM Cloud CLI.
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:67]:
```suggestion
1. Log in into your cluster using `oc` CLI or IBM Cloud CLI.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+2. Verify that all the worker nodes are healthy.
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:68]:
```suggestion
2. Verify that all the worker nodes have the `Ready` status.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+
+    ```
+    $oc get nodes
+    NAME            STATUS   ROLES           AGE   VERSION
+    169.48.170.83   Ready    master,worker   28h   v1.19.0+3b01205
+    169.48.170.88   Ready    master,worker   28h   v1.19.0+3b01205
+    169.48.170.90   Ready    master,worker   28h   v1.19.0+3b01205
+    ```
+
+3. Create a cluster group.Cluster Group
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:78]:
```suggestion
3. Create a cluster group.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+   - From IBM Cloud Web Console
+     > https://cloud.ibm.com/satellite/clusters -> Cluster groups -> Create cluster group
+   - Add cluster to the group
+     > https://cloud.ibm.com/satellite/groups -> select the cluster group -> Clusters -> Add cluster
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:82]:
```suggestion
   - From the [IBM Cloud Web Console](https://cloud.ibm.com/satellite/clusters), click **Cluster groups** -> **Create cluster group**
   - Add your cluster to the group. From the [groups page](https://cloud.ibm.com/satellite/groups), select your cluster group and click **Clusters** -> **Add cluster.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+  ibmcloud sat storage config create --name ocs-config --template-name ocs-local --template-version 4.7 -p "ocs-cluster-name=testocscluster" -p "mon-storage-class=vpc-custom-10iops-tier" -p "mon-size=50Gi" -p "osd-storage-class=vpc-custom-10iops-tier" -p "osd-size=150Gi" -p "num-of-osd=1" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ocs-upgrade=true" -p "ibm-cos-endpoint=https://s3.us-east.cloud-object-storage.appdomain.cloud" -p "ibm-cos-location=us-east-standard" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy"
+  ```
+
+4. Assign your configuration to your cluster groups.
+
+    ```
+    $ ibmcloud sat storage assignment create --name ocs-sub3 --group test-group2 --config ocs-config3
+    ```
+
+5. Verify that your configuration is updated
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:259]:
```suggestion
5. Verify that your configuration is updated.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 8]
```diff
@@ -0,0 +1,320 @@
...
+      Storage Cluster Status:  Ready
+      workerNodes:
+      - 169.48.170.83
+      - 169.48.170.88
+      - 169.48.170.90
+    status:
+       storageClusterStatus: Ready
+    ```
+
+If the storageClusterStatus is stuck in `Progressing` or if it's `Error`, OCS installation has failed.
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:297]:
```suggestion
If the `storageClusterStatus` is `Progressing` or `Error`, the OCS installation has failed.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+    ```
+    oc patch ocsinitialization ocsinit -n openshift-storage --type json --patch  '[{ "op": "replace", "path": "/spec/enableCephTools", "value": true }]'
+    ```
+
+## Removing your OCS configuration
+
+1) Delete all the assignments and configurations created
+2) Remove the finalizer on the OcsCluster resource and delete it
+3) You'll have to first delete the `openshift-storage` namespace after removing finalizers on the resources under it
+4) run this command for all the nodes to clean-up the files created by OCS.
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:317]:
```suggestion
4) Run the following command on each worker node to clean-up the files created by OCS.
```

**Review by derekpoindexter — APPROVED** [review]

**Comment by derekpoindexter** [comment]
left some commit suggestions @Shirisha-S-Rao-1

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 10]
```diff
@@ -0,0 +1,317 @@
...
+| `osd-size` | Required | Enter the size of the osd pods | 100Gi | string |
+| `num-of-osd` | Optional | Enter the number of OSDs. OCS will create 3x number of OSDs for the value specified. Initial storage capacity is the same as your osd size specified at `osd-size`. When you want to increase your storage capacity, you have to increase `num-of-osd` by the multiples of `osd-size` | 1 | integer |
+|`worker-nodes` | Optional | Enter the IP addresses of the worker nodes where you want to deploy OCS. If you do not specify the `worker-nodes`, OCS is installed on all of the worker nodes in your cluster. The minimum number of worker nodes that you must specify is 3. | N/A |csv |
+| `billing-type` | Optional | Enter the billing option that you want to use. You can enter either `hourly` or `monthly`. | `hourly` | string |
+| `ocs-upgrade` | Optional | Set to `true` if you want to upgrade the major version of OCS while creating a configuration of the newer version. | false | boolean |
+| `ibm-cos-endpoint` | Required | Enter the IBM COS regional public endpoint. Example: `s3.us-east.cloud-object-storage.appdomain.cloud` | N/A | string |
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:47]:
```suggestion
| `ibm-cos-endpoint` | Required | Enter the IBM COS regional public endpoint. Example: `https://s3.us-east.cloud-object-storage.appdomain.cloud` | N/A | string |
```

^ based on the command below this endpoint needs to have `https://`

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 11]
```diff
@@ -0,0 +1,317 @@
...
+    You can get the endpoint details from the list of [IBM COS endpoints](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints) and the location from [IBM COS location constraint details](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-classes).
+
+## Red hat Openshift Container Storage - Remote Storage: Parameter reference
+
+**Description of the template parameters :**
+
+| Parameter | Required? | Description | Default value if not provided | Datatype |
+| --- | --- | --- | --- | --- |
+| `ocs-cluster-name` | Required | Enter the name of your OcsCluster custom resource. | N/A | string |
+| `mon-storage-class` | Required | Enter the storage class name that you want to use for the mon pods | N/A | string |
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:39]:
```suggestion
| `mon-storage-class` | Required | Enter the storage class name that you want to use for the mon pods | N/A | string |
```

^ for the storage class - if the cluster is multizone does it need to be a metro class with waitforfirstconsumer binding mode? - i see the command below uses a custom class

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 12]
```diff
@@ -0,0 +1,317 @@
...
+Run the following commands to create a COS instance and create a set of HMAC credentials.
+
+1. Create an IBM COS Service Instance.
+    ```
+    $ ibmcloud resource service-instance-create noobaa-stor cloud-object-storage standard global
+    ```
+
+2. Create HMAC credentials. Make a note of your credentials.
+    ```
+    $ ibmcloud resource service-key-create cos-cred-rw Writer --instance-name noobaa-stor --parameters '{ "HMAC": true}'
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:26]:
```suggestion
    ibmcloud resource service-key-create cos-cred-rw Writer --instance-name noobaa-stor --parameters '{ "HMAC": true}'
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 13]
```diff
@@ -0,0 +1,317 @@
...
+- The storage class you use for the `mon-storage-class` and `osd-storage-class` parameters should have `VolumeBindingMode` set to `WaitForFirstConsumer`
+- You must provision an instance of IBM Cloud Object Storage and provide your COS HMAC credentials, the regional public endpoint, and the IBM COS location when you create your storage configuration.
+
+### Creating the IBM COS service instance
+
+Run the following commands to create a COS instance and create a set of HMAC credentials.
+
+1. Create an IBM COS Service Instance.
+    ```
+    $ ibmcloud resource service-instance-create noobaa-stor cloud-object-storage standard global
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:21]:
```suggestion
    ibmcloud resource service-instance-create noobaa-stor cloud-object-storage standard global
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 14]
```diff
@@ -0,0 +1,317 @@
...
+    ```
+
+If the `storageClusterStatus` is `Progressing` or `Error`, the OCS installation has failed.
+
+### If your OCS installation fails, complete the following the steps to troubleshoot your deployment.
+1. Check the describe of storagecluster and cephcluster in the openshift-storage namespace and look at the `Events` and the `Status` sections
+
+    ```
+    $ oc describe storagecluster -n openshift-storage
+    $ oc describe cephcluster -n openshift-storage
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:301]:
```suggestion
    oc describe storagecluster -n openshift-storage
    oc describe cephcluster -n openshift-storage
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 15]
```diff
@@ -0,0 +1,317 @@
...
+
+## Removing your OCS configuration
+
+1) Delete all the assignments and configurations created
+2) Remove the finalizer on the OcsCluster resource and delete it
+3) You'll have to first delete the `openshift-storage` namespace after removing finalizers on the resources under it
+4) Run the following command on each worker node to clean-up the files created by OCS.
+```
+oc debug node/<node name> -- chroot /host rm -rvf /var/lib/rook /mnt/local-storage
+```
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:317]:
```suggestion
1. List your storage assignments and find the one that you used for your cluster. 
  ```sh
  ic sat storage assignment ls
  ```

2. Remove the assignment. After the assignment is removed, the driver pods and storage classes are removed from all clusters that were part of the storage assignment. 
  ```sh
  ic sat storage assignment rm --assignment <assignment_name>
  ```


3. Remove your storage assignment.
  ```sh
  ic sat storage assignment rm --assignment <assignment>
  ```


4. Clean up the remaining Kubernetes resources from your cluster. Save the following script in a file called `cleanup.sh` to your local machine.
    ```sh
    #!/bin/bash
    oc delete ns openshift-storage --wait=false
    sleep 20
    kubectl -n openshift-storage patch persistentvolumeclaim/db-noobaa-db-0 -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch cephblockpool.ceph.rook.io/ocs-storagecluster-cephblockpool -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch cephcluster.ceph.rook.io/ocs-storagecluster-cephcluster -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch cephfilesystem.ceph.rook.io/ocs-storagecluster-cephfilesystem -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch cephobjectstore.ceph.rook.io/ocs-storagecluster-cephobjectstore -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch cephobjectstoreuser.ceph.rook.io/noobaa-ceph-objectstore-user -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch cephobjectstoreuser.ceph.rook.io/ocs-storagecluster-cephobjectstoreuser -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch NooBaa/noobaa -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch backingstores.noobaa.io/noobaa-default-backing-store -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch bucketclasses.noobaa.io/noobaa-default-bucket-class -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n openshift-storage patch storagecluster.ocs.openshift.io/ocs-storagecluster -p '{"metadata":{"finalizers":[]}}' --type=merge
    sleep 20
    oc delete pods -n openshift-storage --all --force --grace-period=0
    oc delete ns local-storage --wait=false
    sleep 20
    kubectl -n local-storage patch localvolume.local.storage.openshift.io/local-block -p '{"metadata":{"finalizers":[]}}' --type=merge
    kubectl -n local-storage patch localvolume.local.storage.openshift.io/local-file -p '{"metadata":{"finalizers":[]}}' --type=merge
    sleep 20
    oc delete pods -n local-storage --all --force --grace-period=0
    ```

3. Run the `cleanup.sh` script.
  ```sh
  sh ./cleanup.sh
  ```

4. After you run the cleanup script, log in to each worker node and run the following commands.
  1. Deploy a debug pod and run `chroot /host`.
    ```
    oc debug node/<node name> -- chroot /host
    ```


  2. Run the following command to remove any files or directories on the specified paths.
    ```
    rm -rvf /var/lib/rook /mnt/local-storage
    ```

```
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 16]
```diff
@@ -0,0 +1,317 @@
...
+## Red hat Openshift Container Storage - Remote Storage: Parameter reference
+
+**Description of the template parameters :**
+
+| Parameter | Required? | Description | Default value if not provided | Datatype |
+| --- | --- | --- | --- | --- |
+| `ocs-cluster-name` | Required | Enter the name of your OcsCluster custom resource. | N/A | string |
+| `mon-storage-class` | Required | Enter the storage class name that you want to use for the mon pods | N/A | string |
+| `mon-size` | Required | Enter the size of the mon pods | 20Gi | string |
+| `osd-storage-class` | Required | Enter the storage class name that you want to use for the osd pods | N/A | string |
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:41]:
```suggestion
| `osd-storage-class` | Required | Enter the storage class name that you want to use for the OSD pods. The storage class must have the `waitForFirstConsumer` volume binding mode.  | N/A | string |
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 17]
```diff
@@ -0,0 +1,317 @@
...
+
+Red Hat OpenShift Container Storage is a software-defined storage that is optimised for container environments. It runs as an operator on OpenShift Container Platform to provide highly integrated and simplified persistent storage management for containers.
+
+The user has to provide the input values to the custom resource OcsCluster while creating the satellite configuration to deploy OCS
+
+## Prerequisites
+In order to deploy OCS, the following prerequisites are required.
+- Remote storage available in block mode
+- Your cluster must have a minimum of 3 nodes that each have a minimum 16CPUs and 64GB RAM.
+- Your cluster should be compatible with the OCS version that you're trying to install.
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:11]:
```suggestion
In order to deploy OCS, the following prerequisites are required.

- [Create a Satellite location](cloud.ibm.com/docs/satellite?topic=satellite-locations). 
- [Create a Satellite cluster](cloud.ibm.com/docs/satellite?topic=openshift-satellite-clusters). 
  - Your cluster must have a minimum of 3 worker nodes with at least 16CPUs and 64GB RAM per worker node.
  - Your hosts must meet the [Satellite host requirements](https://cloud.ibm.com/docs/satellite?topic=satellite-host-reqs) in addition to having one of the following remote storage configurations.
    * Two raw devices in block mode that have no partitions or formatted file systems. If your devices have no partitions, each node must have 2 free disks. One disk for the OSD and one disk for the MON.
    * Two raw partitions that have no formatted file system. If your raw devices are partitioned, they must have at least 2 partitions per disk, per worker node.
- [Add your Satellite to a cluster group](cloud.ibm.com/docs/satellite?topic=satellite-cluster-config#setup-clusters-satconfig-groups). 
- Your cluster should be compatible with the OCS version that you're trying to install.
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 18]
```diff
@@ -0,0 +1,317 @@
...
+
+2. Get the configuration details of your ocs-cluster custom resource.
+
+    ```
+    kubectl get ocscluster <ocs-cluster-name>
+    ```
+
+3. Save the configuration details. When you upgrade your OCS version, you must enter the same configuration details and set the `template-version` to the version you want to upgrade to and set the `ocs-upgrade` parameter to `true`.
+  ```
+  ibmcloud sat storage config create --name ocs-config --template-name ocs-local --template-version 4.7 -p "ocs-cluster-name=testocscluster" -p "mon-storage-class=vpc-custom-10iops-tier" -p "mon-size=50Gi" -p "osd-storage-class=vpc-custom-10iops-tier" -p "osd-size=150Gi" -p "num-of-osd=1" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ocs-upgrade=true" -p "ibm-cos-endpoint=https://s3.us-east.cloud-object-storage.appdomain.cloud" -p "ibm-cos-location=us-east-standard" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy"
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:247]:
```suggestion
  ibmcloud sat storage config create --name ocs-config --template-name ocs-remote --template-version 4.7 -p "ocs-cluster-name=testocscluster" -p "mon-storage-class=vpc-custom-10iops-tier" -p "mon-size=50Gi" -p "osd-storage-class=vpc-custom-10iops-tier" -p "osd-size=150Gi" -p "num-of-osd=1" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ocs-upgrade=true" -p "ibm-cos-endpoint=https://s3.us-east.cloud-object-storage.appdomain.cloud" -p "ibm-cos-location=us-east-standard" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy"
```

File: config-templates/redhat/ocs-remote/4.6/README.md [hunk 19]
```diff
@@ -0,0 +1,317 @@
...
+| 150Gi | 1 | 150Gi |
+| 150Gi | 2 | 300Gi |
+| 150Gi | 3 | 450Gi |
+| 200Gi | 1 | 200Gi |
+| 200Gi | 2 | 400Gi |
+| 200Gi | 3 | 600Gi |
+
+1. Create the storage configuration and specify the updated values. In this example, the `num-of-osd` parameter is updated to 2, to double the storage capacity.
+  ```
+  ibmcloud sat storage config create --name ocs-config --template-name ocs-local --template-version 4.6 -p "ocs-cluster-name=testocscluster" -p "mon-storage-class=vpc-custom-10iops-tier" -p "mon-size=50Gi" -p "osd-storage-class=vpc-custom-10iops-tier" -p "osd-size=150Gi" -p "num-of-osd=2" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ibm-cos-endpoint=https://s3.us-east.cloud-object-storage.appdomain.cloud" -p "ibm-cos-location=us-east-standard" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy"
```
Comment (derekpoindexter) [inline config-templates/redhat/ocs-remote/4.6/README.md:202]:
```suggestion
  ibmcloud sat storage config create --name ocs-config --template-name ocs-remote --template-version 4.6 -p "ocs-cluster-name=testocscluster" -p "mon-storage-class=vpc-custom-10iops-tier" -p "mon-size=50Gi" -p "osd-storage-class=vpc-custom-10iops-tier" -p "osd-size=150Gi" -p "num-of-osd=2" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ibm-cos-endpoint=https://s3.us-east.cloud-object-storage.appdomain.cloud" -p "ibm-cos-location=us-east-standard" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy"
```

File: config-templates/redhat/ocs-remote/4.6/README.md (hunk 15 above)
Comment (derekpoindexter) [reply config-templates/redhat/ocs-remote/4.6/README.md:317]: Are the resources in the remote template deployed to the `local-storage` namespace? or is the namespace `remote-storage`?

**Review by derekpoindexter — APPROVED** [review]

File: config-templates/redhat/ocs-remote/4.6/README.md (hunk 15 above)
Comment (Shirisha-S-Rao-1) [reply config-templates/redhat/ocs-remote/4.6/README.md:317]: No, only the openshift-storage namespace is created for the remote template. I'll add only those lines of the script here

File: config-templates/redhat/ocs-remote/4.6/README.md (hunk 17 above)
Comment (Shirisha-S-Rao-1) [reply config-templates/redhat/ocs-remote/4.6/README.md:11]: @derekpoindexter, this is not a requirement for the remote template
```
Two raw devices in block mode that have no partitions or formatted file systems. If your devices have no partitions, each node must have 2 free disks. One disk for the OSD and one disk for the MON.
    * Two raw partitions that have no formatted file system. If your raw devices are partitioned, they must have at least 2 partitions per disk, per worker node
    ```

**Review by derekpoindexter — APPROVED** [review]

**Pull request merged**